x-pack/plugins/ml/server/client/error_wrapper.ts

```diff
@@ -9,9 +9,13 @@ import { ResponseError, CustomHttpResponseOptions } from 'kibana/server';
 
 export function wrapError(error: any): CustomHttpResponseOptions<ResponseError> {
   const boom = isBoom(error) ? error : boomify(error, { statusCode: error.status });
+  const statusCode = boom.output.statusCode;
```

Comment:
This is used for all the server routes - just would like to confirm it works as expected for those and won't have unexpected side effects. From what I can see it looks fine but just want to confirm this was kept in mind when it was updated. Thanks! 😄

Reply:
It's good to question this! i forgot to add a comment about the change I had to make.
I'll update the description now but basically the only way to get additional information in the kibana error response is to add it in an `attributes` object.
The original code set the whole `boom` object as the body, this would be turned into a single string `message`.
From looking through `customError` in `KibanaResponse` and in my tests, it appears these two objects result in the same error message.
```
{
  body: boom,
  ...
}
```
and 
```
{
  body: {
    message: boom,
 }
  ...
}
```
So i used the latter to be able to also add an optional `attributes` object